_examples/kinesis/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/apex/go-apex"
+	"github.com/apex/go-apex/kinesis"
+)
+
+const requestBinURL = "http://requestb.in/14cd4uz1"
```

Comment:
@tj here's the variable I mentioned in https://github.com/apex/go-apex/issues/5. Ideally this could be extracted out of the program and loaded from lambda env or a tag from Kinesis stream.